lmdeploy/turbomind/turbomind.py

```diff
@@ -20,19 +20,20 @@
 from lmdeploy.tokenizer import Tokenizer
 from lmdeploy.utils import _stop_words, get_logger, get_model
 
-from .deploy.converter import (get_model_format, supported_formats,
-                               update_config_weight_type, update_output_format)
+from .deploy.converter import (SUPPORTED_FORMATS,
+                               get_input_model_registered_name,
+                               get_output_model_registered_name_and_config)
 from .deploy.source_model.base import INPUT_MODELS
 from .deploy.target_model.base import OUTPUT_MODELS, TurbomindModelConfig
-from .supported_models import SUPPORTED_ARCHS, get_model_arch, is_supported
+from .supported_models import is_supported
 from .utils import ModelSource, get_model_source
 
 # TODO: find another way import _turbomind
 lmdeploy_dir = osp.split(lmdeploy.__file__)[0]
 sys.path.append(osp.join(lmdeploy_dir, 'lib'))
 import _turbomind as _tm  # noqa: E402
 
-logger = get_logger('lmdeploy')
+logger = get_logger('lmdeploy:turbomind')
 
```

Comment:
with `logger = get_logger('lmdeploy:turbomind')`, when init pipeline like `pipe = pipeline('/nvme/shared/vicuna-7b-v1.5/', log_level='INFO')`, the turbomind ini config doesn't print

Reply:
the logger is rollbacked